nncf/onnx/quantization/quantizer_parameters.py

```diff
@@ -26,15 +25,41 @@ class ONNXQuantizerLayerParameters:
 
     :param scale: Quantizer scale.
     :param zero_point: Quantizer zero point.
-    :param mode: Quantizer mode. Could be Symmetric or Asymmetric.
-    :param axis: Axis for per-channel quantization. Should be none in case of per-tensor.
     :param tensor_type: Signed or Unsigned tensor type.
+    :param axis: Axis for per-channel quantization. Should be none in case of per-tensor.
     """
     scale: np.ndarray
     zero_point: np.ndarray
-    mode: QuantizationMode
+    tensor_type: np.dtype
     axis: Optional[int] = None
-    tensor_type: Optional[np.dtype] = None
+
+
+def convert_fq_params_to_onnx_params(parameters: FakeQuantizeParameters,
+                                     num_bits: int,
+                                     tensor_type: np.dtype,
+                                     axis: Optional[int] = None) -> ONNXQuantizerLayerParameters:
+    """
+    Converts common FakeQuantizeParameters to ONNXQuantizerLayerParameters.
+
+    :param parameters: FakeQuantizeParameters representation.
+    :param num_bits: Number of quantizer bits.
+    :param tensor_type: Value type of the tensor. Could be INT8 or UINT8.
+    :param axis: Axis for per-channel quantization. Should be none in case of per-tensor.
+    :return: Quantizer layer attributes.
+    """
+    levels = parameters.levels
+    if levels not in [255, 256]:
+        raise ValueError('Can only export to INT8/UIN8 256-level ONNX Quantize/Dequantize pairs.')
+
+    input_low, input_high = parameters.input_low, parameters.input_high
+    output_low, output_high = parameters.output_low, parameters.output_high
+    if not np.allclose(input_high, output_high) or not np.allclose(input_low, output_low):
+        raise ValueError('ONNX Quantize/Dequantize pairs only support'
+                         ' input_high == output_high and input_low == output_low.')
+
+    level_low, level_high = get_level_low_level_high(tensor_type, num_bits)
+    scale, zero_point = calculate_scale_zero_point(input_low, input_high, level_low, level_high, levels)
```

Comment:
The signature function is not consistent. I mean levels = level_high - level_low + 1, in your implementation level_low, level_high are onnx parameters, but levels is FQ parameter.

Reply:
Changed `levels` to `narrow_range`